recipes/pyplis/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pyplis" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 06d2e58986efab8bb46a4b5ca27e73790686d83967039732b7746f286e009c66
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - numpy >=1.11
+    - scipy
+    - pandas
+    - astropy
+    - matplotlib >=1.5
+    - basemap
+    - opencv == 3.4.4
```

Comment:
This should be `opencv =3.4.4`

Reply:
And is it really strictly this version number and not `opencv >=3.4.4`

Reply:
@scopatz Thanks, I had problems earlier with newer opencv versions here (circleci checks were failing). I found [this thread](https://github.com/bioconda/bioconda-recipes/issues/13974) where they apparently fixed the opencv version to 3.4.4 (and added jasper as test req.). I will try to leave opencv version open in my next commit, and see if it works as well.